tests/sentry/migrations/test_0817_update_notificationmessage_constraints_for_action_group_open_period.py

```diff
@@ -0,0 +1,176 @@
+from datetime import timedelta
```

Comment:
I'd probably consider moving this into somewhere relevant to `notificationmessage` rather than in the migration tests